### PR TITLE
Fix continuous pickup

### LINKED ISSUE #325: stop_times.continuous_drop_off/continous_pickup incorrectly loaded?
## Observed behavior

A bug here https://github.com/conveyal/gtfs-lib/blob/dev/src/main/java/com/conveyal/gtfs/model/StopTime.java#L93 may prevent the `continuous_pickup`/`continuous_drop_off` fields in the `stop_times` GTFS tables from being imported correctly.

## Expected behavior

The mentioned fields in `stop_times` should be loaded properly.

## Steps to reproduce the problem

Load a GTFS feed with non-default values in the fields mentioned.
Observe that the values in the GTFS editor don't correspond to the values in the GTFS file.

## Notes on configuration and files required to reproduce the issue

datatools-server: any version that contains https://github.com/ibi-group/datatools-server/pull/387
datatools-UI: any version that contains https://github.com/ibi-group/datatools-ui/pull/668

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Closes #325 and adds unit tests to prevent a similar issue from happening again. The test is structured in a way that future GTFS Flex features can also be tested in a similar way (with a new feed that is small enough new features can be added to it)

### CODE REVIEW

File: src/test/java/com/conveyal/gtfs/loader/GtfsFlexTest.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        String zipFileName = TestUtils.zipFolderFiles(
+            "real-world-gtfs-feeds/washington-park-shuttle-with-flex-additions",
+            true);
+        FeedLoadResult feedLoadResult = load(zipFileName, testDataSource);
+        testNamespace = feedLoadResult.uniqueIdentifier;
+        validate(testNamespace, testDataSource);
+    }
+
+    @Test
+    void canImportContinuousPickupAndDropoff() {
```
Comment (br648) [inline src/test/java/com/conveyal/gtfs/loader/GtfsFlexTest.java:39]: Change this to a parameterized test. See https://github.com/conveyal/gtfs-lib/blob/dev/src/test/java/com/conveyal/gtfs/loader/ConditionallyRequiredTest.java

**Review by br648 — CHANGES_REQUESTED** [review]
Just one change recommended to the unit test.

File: src/test/java/com/conveyal/gtfs/loader/GtfsFlexTest.java (hunk 1 above)
Comment (miles-grant-ibigroup) [reply src/test/java/com/conveyal/gtfs/loader/GtfsFlexTest.java]: Excellent suggestion thank you! Refactored in d715be09dfdcf79dbd2e565cbf958359e34eec6b

**Review by br648 — APPROVED** [review]
LGTM!

**Review by binh-dam-ibigroup — APPROVED** [review]
Approving pending figuring out a resolution to the build error.

**Pull request merged**